tests/openbis_functions/test_openbis_interbis.py

```diff
@@ -153,7 +173,8 @@ def test_create_sample_type(sample_code, sample_dict):
 @pytest.mark.login
 @pytest.mark.parametrize("sample, output", [(Constants.testing_sample_name.value, True),
                                             (''.join(random.choice('0123456789ABCDEF') for _ in range(16)), False)])
-def test_exists_in_datastore(setup, sample, output):
-    o = Interbis(Constants.db_url.value)
+def test_exists_in_datastore(setup, sample, output, pytestconfig):
+    chosen_runner = pytestconfig.getoption('--url')
+    o = Interbis(chosen_runner, verify_certificates=False)
 
     assert o.exists_in_datastore(str(sample)) == output
```

Comment:
Now (where we have the openbis docker in the git workflow) we can also include to run the doit task "create_openbis_types" and "upload_to_opnebis" on push with the default localhost openbis instance

Reply:
we upload the samples as of now because the dataset upload does not seem to work with the current docker setup (as of now it has been running for 50min and is still not done, maybe the same issue as the matolab upload)